shell.nix

```diff
@@ -619,6 +621,37 @@ All binaries are for 64-bit operating systems.
    github-release -v edit --tag ${core-tag} --name ${core-tag} --description "$( hc-generate-release-notes )" --pre-release
   '';
 
+  build-release-artifact = path:
+  ''
+   export artifact_name=`sed "s/unknown/generic/g" <<< "${path}-${core-version}-${linux-release-target}"`
+   echo
+   echo "building $artifact_name..."
+   echo
+
+   CARGO_INCREMENTAL=0 cargo rustc --manifest-path ${path}/Cargo.toml --target ${linux-release-target} --release -- -C lto
+   mkdir -p dist/$artifact_name
+   cp target/${linux-release-target}/release/hc ${path}/LICENSE ${path}/README.md dist/$artifact_name
```

Comment:
@samrose experienced an issue, that we traced back to this line, and the fact that the `cp` command hardcodes `hc` into the path, when it also needs to accomodate `holochain` for the conductor binary. @thedavidmeister we are hoping you'll use your nix foo to resolve. 

Nevertheless, we got Sam unblocked by just changing the line by hand, to get the binaries built out

Reply:
yes yes i have a solution i will push it sometime today